client/src/components/Rooms/MeetingRooms/MeetingRooms.css

```diff
@@ -1,22 +1,25 @@
-body {
+/* body {
 	background-image: none;
 	background-color: #202541;
 	color: white;
-}
+} */
 
 .meetingRooms {
 	width: 80%;
 	display: flex;
 	flex-direction: column;
 	margin: auto;
+	/* background-image: none;
```

Comment:
Looks good overall but we should remove commented code if not required :)

Reply:
will do